### PR TITLE
Fixing Wrong Examples

### PR DESCRIPTION
**Pull request by TunayAdaKaracan**
I fixed some wrong examples i found.

### CODE REVIEW

File: docs/Voice/receiving.mdx [hunk 1]
```diff
@@ -109,7 +110,6 @@ async def stop_recording(ctx):
     if ctx.guild.id in connections:  # Check if the guild is in the cache.
         vc = connections[ctx.guild.id]
         vc.stop_recording()  # Stop recording, and call the callback (once_done).
-        del connections[ctx.guild.id]  # Remove the guild from the cache.
```
Comment (BruceCodesGithub) [inline docs/Voice/receiving.mdx:112]: Why is this part removed?

Comment (TunayAdaKaracan) [reply docs/Voice/receiving.mdx:112]: I re-added it on "once" function. The reason why i moved it bcz of "once" function is async. So when we call it and delete dict, dict is removed before using "await connections[channel.guild.id].disconnect()".

Comment (TunayAdaKaracan) [reply docs/Voice/receiving.mdx:112]: Oh i noticed that, i forget something to add!

**Comment by TunayAdaKaracan** [comment]
I figured out, that i mistaken something in vc. so i restored it.

**Review by Dorukyum — APPROVED** [review]

**Review by VincentRPS — APPROVED** [review]
